curriculum/challenges/spanish/04-data-visualization/data-visualization-projects/visualize-data-with-a-heat-map.spanish.md

```diff
@@ -4,11 +4,11 @@ title: Visualize Data with a Heat Map
er un eje y con una <code>id=&quot;y-axis&quot;</code> correspondiente <code>id=&quot;y-axis&quot;</code> . <strong>Historia de usuario n. ° 5:</strong> Mi mapa de calor debe tener elementos <code>rect</code> con una <code>class=&quot;cell&quot;</code> que representa los datos. <strong>Historia de usuario n. ° 6:</strong> Debe haber al menos 4 colores de relleno diferentes utilizados para las celdas. <strong>Historia de usuario n. ° 7:</strong> Cada celda tendrá las propiedades <code>data-month</code> , <code>data-year</code> , <code>data-temp</code> contienen sus valores correspondientes de mes, año y temperatura. <strong>Historia de usuario n. ° 8:</strong> El <code>data-month</code> <code>data-year</code> de <code>data-year</code> de cada celda debe estar dentro del rango de los datos. <strong>Historia de usuario n. ° 9:</strong> Mi mapa de calor debe tener celdas que se alineen con el mes correspondiente en el eje y. <strong>Historia de usuario # 10:</strong> Mi mapa de calor debe tener celdas que se alineen con el año correspondiente en el eje x. <strong>Historia de usuario n. ° 11:</strong> Mi mapa de calor debe tener varias etiquetas de tic en el eje y con el nombre completo del mes. <strong>Historia de usuario n. ° 12:</strong> Mi mapa de calor debe tener varias etiquetas de marca en el eje x con los años comprendidos entre 1754 y 2015. <strong>Historia de usuario n. ° 13:</strong> Mi mapa de calor debe tener una leyenda con una <code>id=&quot;legend&quot;</code> correspondiente <code>id=&quot;legend&quot;</code> . <strong>Historia de usuario n. ° 14:</strong> Mi leyenda debe contener elementos <code>rect</code> . <strong>Historia de usuario n. ° 15:</strong> Los elementos <code>rect</code> en la leyenda deben usar al menos 4 colores de relleno diferentes. <strong>Historia de usuario n. ° 16:</strong> puedo pasar el mouse sobre un área y ver una información sobre herramientas con la correspondiente <code>id=&quot;tooltip&quot;</code> que muestra más información sobre el área. <strong>Historia de usuario n. ° 16:</strong> Mi información sobre herramientas debe tener una propiedad de <code>data-year</code> que corresponda con el <code>data-year</code> de <code>data-year</code> del área activa. Aquí está el conjunto de datos que necesitará para completar este proyecto: <code>https://raw.githubusercontent.com/freeCodeCamp/ProjectReferenceData/master/global-temperature.json</code> Puede construir su proyecto por medio de <a href="https://codepen.io/freeCodeCamp/pen/MJjpwO" target="_blank">este bolígrafo CodePen</a> . O puede usar este enlace CDN para ejecutar las pruebas en el entorno que desee: <code>https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js</code> Una vez que haya terminado, envíe la URL a su cuenta Proyecto con todas sus pruebas aprobadas. Recuerda usar el método de <a href="https://forum.freecodecamp.org/t/how-to-get-help-when-you-are-stuck/19514" target="_blank">lectura-búsqueda-pregunta</a> si te atascas. </section>
```

Comment:
The user stories sound more confusing this way. I suggest you drop the left hand side of the equations if you don't like the previous versions, eg. 
`Mi mapa de calor debe tener un título con una "id" de <code>&quot;title&quot;</code> `

Original:
https://github.com/freeCodeCamp/freeCodeCamp/blob/master/curriculum/challenges/english/04-data-visualization/data-visualization-projects/visualize-data-with-a-heat-map.english.md